packages/portalnetwork/src/client/constructor.ts

```diff
@@ -100,7 +100,7 @@ export async function createPortalNetwork(opts: Partial<PortalNetworkOpts>): Pro
       break
     }
     case TransportLayer.MOBILE:
```

Comment:
```suggestion
    case TransportLayer.TAURI:
```
Let's add `TAURI` to the `TransportLayer` enum and use this case instead of `MOBILE`